Cargo.toml

```diff
@@ -3,21 +3,21 @@ authors = ["Antoni Boucher <bouanto@zoho.com>"]
 description = "Rust bindings for the VTE 3 library"
 license = "MIT"
 name = "vte-rs"
-version = "0.2.0"
+version = "0.3.0"
 
 [dependencies]
 bitflags = "1"
-gdk-sys = { git="https://github.com/gtk-rs/sys" }
-gdk = { git="https://github.com/gtk-rs/gdk" }
-gio = { git="https://github.com/gtk-rs/gio" }
-gio-sys = { git="https://github.com/gtk-rs/sys" }
-glib = { git="https://github.com/gtk-rs/glib" }
-glib-sys = { git="https://github.com/gtk-rs/sys" }
-gobject-sys = { git="https://github.com/gtk-rs/sys" }
-gtk = { git="https://github.com/gtk-rs/gtk" }
-gtk-sys = { git="https://github.com/gtk-rs/sys" }
+gdk-sys = ""
```

Comment:
Why not writing the version of dependencies?

Reply:
That way we don't need to change every single one on upgrades. This way cargo will simply choose the right crates to go along with the gtk-sys package we've chosen.

Reply:
Does that mean we cannot publish the crate, though?
In any case, I don't mind either way.

Reply:
Oh, didn't know that'd cause that. I'll add the versions then, thanks.